src/app/[locale]/dataset-list.tsx

```diff
@@ -0,0 +1,38 @@
+'use client';
+
+import {useQuery} from '@tanstack/react-query';
+import {SearchResult} from '@/lib/dataset-fetcher';
+import DatasetCard from '@/components/dataset-card';
```

Comment:
At this point only on this page, the same as the component FilterSet. I like extracting pure layout components from the logic. But I can also place it in this file as a separate component.

Reply:
Good one! I prefer to put the component in `src/app/[locale]` rather than in `src/components/dataset-card`, to make clear that it (currently) is a component that belongs to a specific page. This way we're utilizing the new [colocating concept](https://beta.nextjs.org/docs/routing/fundamentals#colocation) of Next 13.

Reply:
👍